git/pre-commit

```diff
@@ -2,6 +2,14 @@
 
 if [ "$NOLINT" ]; then exit 0; fi
 
+if [ "$TRAVIS" != true ]; then
```

Comment:
@tomeyro 
Could you consider the following cases?

| NOLINT | Ask before of run lints? | 
|-----------|------|
| Undefined       |  yes |
| 1       |  exit early (it is) |
| 0        | No  |